### ISSUE TITLE
Travis CI failing on slycot builds

### ISSUE THREAD

**Issue by murrayrm** [python-control/python-control#277]
PR #276 Travis CI checks failed due to an issue with the slycot builds.  See [Travis CI build 400.01, line 954](https://travis-ci.org/python-control/python-control/jobs/497800396). I did a quick check and it looks like there is a problem with `skbuild`:
```
Traceback (most recent call last):
  File "setup.py", line 8, in <module>
    from skbuild import setup
ModuleNotFoundError: No module named 'skbuild'
0.03s$ if [ $SLYCOT != "" ]; then python -c "import slycot"; fi
```
Not sure if this is something transient or something that has changed in `scikit` (and not quite sure why `slycot` needs `scikit`).

**Comment by repagh** [comment]
Building Slycot needs scikit-build, and since the check does that first ...
see PR #280

**Comment by murrayrm** [comment]
Fixed in PR #280
